lib/public-stats.js

```diff
@@ -348,19 +348,20 @@ export const updateDailyAllocatorRetrievalStats = async (pgClient, committees, f
 /**
 * @param {Iterable<Committee>} committees
 * @param {(minerId: string, cid: string) => (string[] | undefined)} findDealActors
-* @param {string} actorName
+* @param {'client' | 'allocator'} partyName
 * @returns {object[]}
 */
-function buildPerActorStats (committees, findDealActors, actorName) {
-  const retrievalStatsPerActor = new Map()
+function buildPerPartyStats (committees, findDealActors, partyName) {
```

Comment:
Let's rename "actor" to "party" everywhere, please.

```suggestion
function buildPerPartyStats (committees, findDealParties, partyName) {
```